tests/testthat/test_read_emigration.R

```diff
@@ -26,7 +27,7 @@ test_that("read_emigration", {
 
   # add labels
   test4 <- read_emigration(add_labels = 'pt', columns = c('abbrev_state', 'V1006'))
-  test4 <- test4 |> filter(abbrev_state == 'CE') |> as.data.frame()
+  test4 <- test4 |> dplyr::filter(abbrev_state == 'CE') |> as.data.frame()
```

Comment:
I was running individual files to test and `dplyr` wasn't loaded.

Reply:
nice catch